spec/models/generic_file_spec.rb

```diff
@@ -75,6 +70,7 @@
   describe '#append_metadata', :integration => true do
 
     before  do
+      # TODO: Don't use a before each, use a let instead
```

Comment:
This appears to be a common issue in our test suite, is using a before each block to define objects for our test cases. Should be using a let block instead as it's faster performance wise. Let blocks are lazily loaded and only used when referenced.